src/core/lib/gprpp/arena.h

```diff
@@ -76,7 +83,9 @@ class Arena {
     Zone* prev;
   };
 
-  explicit Arena(size_t initial_size) : initial_zone_size_(initial_size) {}
+  explicit Arena(size_t initial_size, size_t initial_alloc = 0)
+      : total_used_(initial_alloc), initial_zone_size_(initial_size) {}
```

Comment:
Nit: Can we use the same order for the ctor parameters and the data members?

Reply:
Unable - the initial_alloc parameter must be optional, so it must be second. But for performance reasons (for Alloc()) it's better to have total_user_  be the first element.